tests/frontend/travis/runnerBackend.sh

```diff
@@ -31,20 +31,12 @@ echo "Successfully connected to Etherpad on http://localhost:9001"
 # a copy of settings.json is necessary for the backend tests to work
 cp settings.json.template settings.json
 
-# Build the minified files?
-curl http://localhost:9001/p/minifyme -f -s > /dev/null
```

Comment:
why remove this?

Reply:
shouldn't be necessary because api- and contentcollector-tests both include files directly. minification is only relevant for frontend tests, no?